libs/wingc/src/type_check.rs

```diff
@@ -569,6 +569,8 @@ impl Subtype for Type {
 					return false;
 				}
 
+				// ^ example phase check
```

Comment:
This is most likely where we fail the signature comparison between two identical functions with different phases. This however isn't the best of places to add the diagnostic message since whoever is calling this will probably add their own diagnostic upon a false result. You can get pretty far understanding this by just [running it under a debugger and putting a breakpoint here](https://www.winglang.io/contributing/start-here/development#how-do-i-debug-the-wing-compiler-on-vscode).

Reply:
Yeah thought so too, just noted this as a way how to check the phase as the codebase is new to me 😅. Thanks for the link regarding debugger, gotta check it out.